Sources/Sentry/SentryBreadcrumbStore.m

```diff
@@ -1,77 +0,0 @@
-//
-//  SentryBreadcrumbStore.m
-//  Sentry
-//
-//  Created by Daniel Griesser on 22/05/2017.
-//  Copyright © 2017 Sentry. All rights reserved.
-//
-
-#if __has_include(<Sentry/Sentry.h>)
-
-#import <Sentry/SentryBreadcrumbStore.h>
-#import <Sentry/SentryBreadcrumb.h>
-#import <Sentry/SentryLog.h>
-#import <Sentry/SentryFileManager.h>
-
-#else
-#import "SentryBreadcrumbStore.h"
-#import "SentryBreadcrumb.h"
-#import "SentryLog.h"
-#import "SentryFileManager.h"
-#endif
-
-NS_ASSUME_NONNULL_BEGIN
-
-@interface SentryBreadcrumbStore ()
-
-@property(nonatomic, strong) SentryFileManager *fileManager;
-
-@end
-
-@implementation SentryBreadcrumbStore
-
-- (instancetype)initWithFileManager:(SentryFileManager *)fileManager {
-    self = [super init];
-    if (self) {
-        self.maxBreadcrumbs = 50;
-        self.fileManager = fileManager;
-    }
-    return self;
-}
-
-- (void)addBreadcrumb:(SentryBreadcrumb *)crumb {
-    [SentryLog logWithMessage:[NSString stringWithFormat:@"Add breadcrumb: %@", crumb] andLevel:kSentryLogLevelDebug];
-    [self.fileManager storeBreadcrumb:crumb maxCount:self.maxBreadcrumbs];
```

Comment:
Previously, only 50 breadcrumbs were stored and they were saved into the filemanager (which presumably stores things on disk).